### PR TITLE
Added section on plugin installation

### PR DESCRIPTION
**Pull request by unrealapex**

### CODE REVIEW

**Comment by unrealapex** [comment]
@junegunn would you please be able to review this request.

**Comment by junegunn** [comment]
Can you move the section to above `Usage` similar to https://github.com/junegunn/goyo.vim?tab=readme-ov-file#installation?

File: README.md [hunk 1]
```diff
@@ -18,6 +29,8 @@ Usage
 - `Limelight!! [0.0 ~ 1.0]`
     - Toggle Limelight
 
+
+
```
Comment (junegunn) [inline README.md:33]:
```suggestion
```

**Comment by junegunn** [comment]
Thanks.

**Pull request merged**